extensions/amp-script/validator-amp-script.protoascii

```diff
@@ -204,6 +204,11 @@ tags: {  # <amp-script>
     }
     blacklisted_value_regex: "__amp_source_origin"
   }
+  attrs: {
+    name: "development"
+    deprecation: "The 'data-ampdevmode' attribute on the root html element"
+    deprecation_url: "https://github.com/ampproject/amphtml/issues/24987"
```

Comment:
Does this mean the field is still invalid? We can't allow pages with the attribute to be cached.

Reply:
This means the attribute would be allowed (valid) until it is removed. It produces a [warning](https://github.com/ampproject/amphtml/blob/master/validator/validator.proto#L144).

Since it wasn't allowed in validation, perhaps this does not need a I2D.

Reply:
Thats a good point! By marking it deprecated I'm actually making it valid. I'll remove the deprecation notice. I'm not sure what this means for the I2D (is this still considered a feature being deprecated?).